File: beecrowd/1048/1048 - Salary Increase .py
oSalary = float(input())

if 0 <= oSalary <= 400:
    rate = 15
elif 400.01 <= oSalary <= 800:
    rate = 12
elif 800.01 <= oSalary <= 1200:
    rate = 10
elif 1200.01 <= oSalary <= 2000:
    rate = 7
elif oSalary > 2000:
    rate = 4
else:
    rate = 0

nSalary = oSalary + ((oSalary * rate) / 100)
mEarned = nSalary - oSalary

print(f"Novo salario: {nSalary:.2f}")
print(f"Reajuste ganho: {mEarned:.2f}")
print("Em percentual:", rate, "%")

File: beecrowd/1048/1048 - Salary Increase .cs
using System;

class Program
{
	static void Main(string[] args)
	{
		double oSalary, nSalary, mEarned;
		int rate;

		oSalary = double.Parse(Console.ReadLine());

		if (oSalary >= 0 && oSalary <= 400)
			rate = 15;
		else if (oSalary >= 400.01 && oSalary <= 800)
			rate = 12;
		else if (oSalary >= 800.01 && oSalary <= 1200)
			rate = 10;
		else if (oSalary >= 1200.01 && oSalary <= 2000)
			rate = 7;
		else if (oSalary > 2000)
			rate = 4;
		else
			rate = 0;

		nSalary = oSalary + ((oSalary * rate) / 100);
		mEarned = nSalary - oSalary;

		Console.WriteLine($"Novo salario: {nSalary:F2}");
		Console.WriteLine($"Reajuste ganho: {mEarned:F2}");
		Console.WriteLine("Em percentual: " + rate + " %");

	}
}